docs/install.md

```diff
@@ -30,23 +30,26 @@ To install Compose, do the following:
 
 3. Go to the <a href="https://github.com/docker/compose/releases" target="_blank">Compose repository release page on GitHub</a>.
 
-4. Follow the instructions from the release page and run the `curl` command,
+4. Follow the instructions from the release page and run the `wget` or `curl` command below,
 which the release page specifies, in your terminal.
 
-     > Note: If you get a "Permission denied" error, your `/usr/local/bin` directory
-     probably isn't writable and you'll need to install Compose as the superuser. Run
-     `sudo -i`, then the two commands below, then `exit`.
-
      The following is an example command illustrating the format:
 
-        curl -L https://github.com/docker/compose/releases/download/1.6.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+        $ LATEST_TAG=$(curl -s 'https://api.github.com/repos/docker/compose/tags?page=1&per_page=1' | python -c 'import sys, json; print json.load(sys.stdin)[0]["name"]')
```

Comment:
unfortunately, it will...

just a idea, can it be delivered via script?
just like how we setup docker, `$ curl -sSL https://get.docker.com/ | sudo sh -`